drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -117,26 +117,13 @@ static String get_constant_text(SL::DataType p_type, const Vector<SL::ConstantNo
 			return text.as_string();
 		}
 
-		// GLSL ES 2 doesn't support uints, so we just use signed ints instead...
-		case SL::TYPE_UINT: return itos(p_values[0].uint);
```

Comment:
Wasn't this code meant to actually make `uint` and `uvec*` work? It was not working?

Reply:
I think its redundant... but want to ask @reduz if this can be deleted

Reply:
Nevermind, I've brought it back...

Reply:
Well I don't know about it, it just reads like this code is meant to make `uint` actually *work* in GLES2 shaders, so that it wouldn't have to be excluded.

But your tests seem to show that it's actually not working, so either this code was buggy, or it was not working? Not sure.

Reply:
I can remove it in other PR, later after 3.2(so that not cause possible regression). I don't think it's so important if the uint is not allowed - it's not generated and this code is not reached at all.